src/sage/libs/flint/acb_macros.pxd

```diff
@@ -0,0 +1,11 @@
+# Macros from acb.h
+# See https://github.com/flintlib/flint/issues/152
```

Comment:
wrong issue number? (should be 1529, I think)

Reply:
fixed by you!